dds/DCPS/transport/rtps_udp/MetaSubmessage.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#include "MetaSubmessage.h"
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace OpenDDS {
+namespace DCPS {
+
+bool MetaSubmessage::operator==(const MetaSubmessage& other) const
```

Comment:
This is for use in tests only, correct? Any chance we can move it to a test utility directory so we know it's never getting used during production?

Reply:
Yes and yes.